[2025-06-09T07:53:04.090+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-09T07:53:04.117+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: netsuite_saved_search_4532_memory.extract_from_netsuite manual__2025-06-09T07:47:54.915806+00:00 [queued]>
[2025-06-09T07:53:04.125+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: netsuite_saved_search_4532_memory.extract_from_netsuite manual__2025-06-09T07:47:54.915806+00:00 [queued]>
[2025-06-09T07:53:04.125+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-06-09T07:53:04.138+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_from_netsuite> on 2025-06-09 07:47:54.915806+00:00
[2025-06-09T07:53:04.144+0000] {standard_task_runner.py:63} INFO - Started process 45 to run task
[2025-06-09T07:53:04.147+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'netsuite_saved_search_4532_memory', 'extract_from_netsuite', 'manual__2025-06-09T07:47:54.915806+00:00', '--job-id', '369', '--raw', '--subdir', 'DAGS_FOLDER/netsuite_4532_dag.py', '--cfg-path', '/tmp/tmpni3h9t43']
[2025-06-09T07:53:04.150+0000] {standard_task_runner.py:91} INFO - Job 369: Subtask extract_from_netsuite
[2025-06-09T07:53:04.199+0000] {task_command.py:426} INFO - Running <TaskInstance: netsuite_saved_search_4532_memory.extract_from_netsuite manual__2025-06-09T07:47:54.915806+00:00 [running]> on host 46960f15b31b
[2025-06-09T07:53:04.281+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='netsuite_saved_search_4532_memory' AIRFLOW_CTX_TASK_ID='extract_from_netsuite' AIRFLOW_CTX_EXECUTION_DATE='2025-06-09T07:47:54.915806+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-09T07:47:54.915806+00:00'
[2025-06-09T07:53:04.282+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-09T07:53:06.663+0000] {logging_mixin.py:188} INFO - üîç Login status code: 200
[2025-06-09T07:53:06.663+0000] {logging_mixin.py:188} INFO - üîç Content-Type: text/html;charset=utf-8
[2025-06-09T07:53:06.664+0000] {logging_mixin.py:188} INFO - üîç Response sample: <!DOCTYPE html>
<html data-scroll-behavior="natural" lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
	<script type="importmap">
		{"imports": {"@uif-js/builder":"/assets/@uif-js/builder/7.0.0/min/in
[2025-06-09T07:53:06.664+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-09T07:53:06.665+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/netsuite_4532_dag.py", line 61, in extract_from_netsuite
    raise Exception("‚ùå ƒêƒÉng nh·∫≠p NetSuite th·∫•t b·∫°i ho·∫∑c tr·∫£ v·ªÅ HTML")
Exception: ‚ùå ƒêƒÉng nh·∫≠p NetSuite th·∫•t b·∫°i ho·∫∑c tr·∫£ v·ªÅ HTML
[2025-06-09T07:53:06.679+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=netsuite_saved_search_4532_memory, task_id=extract_from_netsuite, run_id=manual__2025-06-09T07:47:54.915806+00:00, execution_date=20250609T074754, start_date=20250609T075304, end_date=20250609T075306
[2025-06-09T07:53:06.691+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 369 for task extract_from_netsuite (‚ùå ƒêƒÉng nh·∫≠p NetSuite th·∫•t b·∫°i ho·∫∑c tr·∫£ v·ªÅ HTML; 45)
[2025-06-09T07:53:06.734+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-06-09T07:53:06.774+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-09T07:53:06.778+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
